components/widgets/users/avatar/avatar.scss

```diff
@@ -55,6 +55,20 @@
         font-size: 18px;
     }
 
+    &.Avatar-xl-custom-GM {
+        width: 72px;
+        min-width: 72px;
+        height: 72px;
+        font-size: 18px;
+    }
+
+    &.Avatar-xl-custom-DM {
+        width: 96px;
+        min-width: 96px;
+        height: 96px;
+        font-size: 18px;
+    }
+
```

Comment:
@matthewbirtch do we really need new variants here ... IMO we should just use the existing ones

Reply:
@michelengelen I don't know what sizes we have available in the existing ones. If these sizes already exist (or if we have sizes close to this), then I'm fine with using the existing.